File: Suiki/app/src/main/java/com/suiki/suiki/Model/HttpModel/HttpReponse.java
package com.suiki.suiki.Model.HttpModel;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Massil on 27/01/2017.
 */

public class HttpReponse {
    @SerializedName("status")
    public boolean status;
    @SerializedName("data")
    public Object data;
    @SerializedName("error")
    public Object error;
}

File: Service/model/http/httpReponse.js
/**
 * Created by Massil on 27/01/2017.
 */
var reponse = {
    status : 'String',
    data : 'Object'
};

module.exports = {
    setReponse : function(status , data , err)
    {
        reponse.data = data;
        reponse.status = status;
        reponse.error = err;
        return reponse;
    },
    getReponse : function()
    {
        return reponse;
    }
};